diff --git a/dist/plugin-B4MlD0Bd.js b/dist/plugin-B4MlD0Bd.js
index 7ddbbcf1630129b133dbf3e8621942478b5348b8..d0c8757b35dfddaae3d7ec4de4d164fca392d97c 100644
--- a/dist/plugin-B4MlD0Bd.js
+++ b/dist/plugin-B4MlD0Bd.js
@@ -1,5 +1,4 @@
 import assert from "node:assert/strict";
-import { isCSSRequest, normalizePath } from "vite";
 import fs from "node:fs";
 import path from "node:path";
 import { fileURLToPath } from "node:url";
@@ -90,6 +89,13 @@ function slash(p) {
 	return p.replace(windowsSlashRE, "/");
 }
 const isWindows = typeof process !== "undefined" && process.platform === "win32";
+function normalizePath(id) {
+	return path.posix.normalize(isWindows ? slash(id) : id);
+}
+const CSS_LANGS_RE = /\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)(?:$|\?)/;
+function isCSSRequest(request) {
+	return CSS_LANGS_RE.test(request);
+}
 function injectQuery(url, queryToInject) {
 	const { file, postfix } = splitFileAndPostfix(url);
 	return `${isWindows ? slash(file) : file}?${queryToInject}${postfix[0] === "?" ? `&${postfix.slice(1)}` : postfix}`;
@@ -502,7 +508,8 @@ function collectAssetDepsInner(fileName, bundle) {
 	};
 }
 function patchViteClientPlugin() {
-	const viteClientPath = normalizePath(fileURLToPath(import.meta.resolve("vite/dist/client/client.mjs")));
+	let viteClientPath
+	try { viteClientPath = normalizePath(fileURLToPath(import.meta.resolve("vite/dist/client/client.mjs"))); } catch { return null; }
 	function endIndexOf(code, searchValue) {
 		const i = code.lastIndexOf(searchValue);
 		return i === -1 ? i : i + searchValue.length;
@@ -514,8 +521,12 @@ function patchViteClientPlugin() {
 			handler(code, id) {
 				if (id === viteClientPath) {
 					if (code.includes("linkSheetsMap")) return;
+					const sheetsMapIndex = code.indexOf("const sheetsMap");
+					const updateStyleIndex = endIndexOf(code, "function updateStyle(id, content) {");
+					const removeStyleIndex = endIndexOf(code, "function removeStyle(id) {");
+					if (sheetsMapIndex === -1 || updateStyleIndex === -1 || removeStyleIndex === -1) return;
 					const s = new MagicString(code);
-					s.prependLeft(code.indexOf("const sheetsMap"), `\
+					s.prependLeft(sheetsMapIndex, `\
 const linkSheetsMap = new Map();
 document
   .querySelectorAll('link[rel="stylesheet"][data-vite-dev-id]')
@@ -523,8 +534,8 @@ document
     linkSheetsMap.set(el.getAttribute('data-vite-dev-id'), el)
   });
 `);
-					s.appendLeft(endIndexOf(code, `function updateStyle(id, content) {`), `if (linkSheetsMap.has(id)) { return }`);
-					s.appendLeft(endIndexOf(code, `function removeStyle(id) {`), `
+					s.appendLeft(updateStyleIndex, `if (linkSheetsMap.has(id)) { return }`);
+					s.appendLeft(removeStyleIndex, `
 const link = linkSheetsMap.get(id);
 if (link) {
   document
